### PR TITLE
[Merged by Bors] - systest: longevity testing

### PR DESCRIPTION
**Pull request by dshulyak**
- adding a set of isolated tests

Initial set includes TestStepShortDisconnect, TestStepTransactions, TestStepReplaceNodes, TestStepVerifyConsistency, TestStepVerifySynced. Tasks should not directly affect the state of the other tasks, in order to enable more flexible workflows.

Some tasks may not be executable in parallel with some other tasks. This is a responsibility of the scheduler to coordinate such tasks.

- manual workflow

1. Create a cluster

```bash
export namespace=yyy
make run test_name=TestStepCreate size=10 bootstrap=1m keep=true
```

2. Run transactions

```bash
make run test_name=TestStepTransactions
```

3. Run replace nodes

```bash
make run test_name=TestStepReplaceNodes
```

4. Run verification

```bash
make run test_name=TestStepVerifySynced
```

- executing scheduler as a test

```bash
make run test_name=TestScheduleBasic size=10 bootstrap=1m keep=true
```

- cluster deployment

every separate smesher is deployed as a separate statefulset. it enables:
1. deleting any smesher in the cluster, not only the last one
2. using new version for smeshers that are added into the cluster after initial deployment 

- observability

1. enabled metrics collection
2. enabled pprof server 

- makefile changes

1. test pod will have a timestamp in the name
it is now possible to execute tasks from the same commit consecutively without running `make clean` after each task.
2. `make cleanall` command to remove all namespace, including ones with `keep` option

### CODE REVIEW

File: sql/database.go [hunk 1]
```diff
@@ -263,6 +263,14 @@ func (tx *Tx) begin(initstmt string) error {
 	return nil
 }
 
+func (tx *Tx) immediate() error {
+	return tx.begin("BEGIN IMMEDIATE;")
```
Comment (countvonzero) [inline sql/database.go]: tx.begin(beginImmediate)

File: sql/database.go [hunk 2]
```diff
@@ -263,6 +263,14 @@ func (tx *Tx) begin(initstmt string) error {
...
+}
+
+func (tx *Tx) deferred() error {
+	return tx.begin("BEGIN;")
```
Comment (countvonzero) [inline sql/database.go]: tx.begin(beginDefault)

**Review by countvonzero — APPROVED** [review]
please update README with the instructions to run this. thanks

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors try

**Comment by dshulyak** [comment]
bors merge

**Comment by dshulyak** [comment]
bors merge

**Pull request closed without merging**